spec/requests/locations_controller_spec.rb

```diff
@@ -264,6 +269,29 @@
           expect(page).to have_content("Check your answers before creating this scheme")
         end
       end
+
+      context "when postcodes.io doesn't return a la_code" do
+        let!(:params) do
```

Comment:
Don't think you need the bang here since you use params in the request?

Reply:
Ah, thanks!